### PR TITLE
MRG: Update version to 0.5.0

### PR DESCRIPTION
**Pull request by davidh-ssec**
- [x] Update doc git submodule once https://github.com/vispy/vispy-website/pull/10 is approved

### CODE REVIEW

**Comment by davidh-ssec** [comment]
Yay flake8 got an update today and changed rules so now the tests fail.

**Comment by davidh-ssec** [comment]
@larsoner Assume the tests pass, could you please review the `test_multiplication` test in `vispy/visuals/transforms/tests/test_transforms.py`? The test function had a typo so it was never actually being run. I fixed the typo and the test failed. I made the tests pass and added a couple semi-redundant tests to be sure that things were working as expected. However, there were a few that didn't seem to make sense and I'm hoping this is just because the transform system changed so much between this version and others. For example it had a test where a chained transform multiplied with a MatrixTransform would produce a MatrixTransform. That doesn't really make sense and sounds like a complicated feature to get working.

File: vispy/visuals/transforms/tests/test_transforms.py [hunk 1]
```diff
@@ -43,13 +43,19 @@ def tesst_multiplication():
...
-    assert_chain_types(s * a * p, [PT, AT])
+    assert_chain_types(s * a * p, [AT, PT])
+    assert_chain_types(p * s * a, [PT, ST, AT])
     assert_chain_types(s * p * s, [ST, PT, ST])
-    assert_chain_types(s * a * p * s * a, [AT, PT, AT])
-    assert_chain_types(c2 * a, [AT])
-    assert_chain_types(p * l * s, [ST, LT, PT])
+    assert_chain_types(s * a * p * s * a, [AT, PT, ST, AT])
+    assert_chain_types(c2 * a, [ST, AT, ST, AT])
+    assert_chain_types(p * log_trans * s, [PT, LT, ST])
```
Comment (larsoner) [inline vispy/visuals/transforms/tests/test_transforms.py]: yeah this looks okay to me

**Comment by kmuehlbauer** [comment]
FYI: I restarted the one failing travis build and it got green lights now. :+1:

**Comment by davidh-ssec** [comment]
@larsoner Yeah I wanted to do it "right" and then I regretted my decision half way through.

**Pull request merged**